lib/neo4j/migrations/helpers.rb

```diff
@@ -0,0 +1,50 @@
+module Neo4j
+  module Migrations
+    module Helpers
+      def rename_property(label, old_property, new_property)
+        by_label(label).where(n: {new_property => nil})
```

Comment:
I don't know that I would expect that this would only work on nodes where the `new_property` is `nil`.

My first instinct was to just blow away any values which are on the `new_property`, but perhaps we should run a check to see if the `new_property` is non-null anywhere and give an error if that's the case.